tests/integration/cli_transfer_token_negative_tes.go

```diff
@@ -0,0 +1,47 @@
+//go:build integration
+
+package integration
+
+import (
+	"fmt"
+	"github.com/canow-co/cheqd-node/tests/integration/cli"
+	"github.com/canow-co/cheqd-node/tests/integration/testdata"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("cheqd cli - negative transfert token", func() {
+	It("cannot transfer token with missing cli arguments", func() {
+		amount := fmt.Sprintf("%dncheq", 100)
```

Comment:
Could you please create a constant for the coin name?

Reply:
Done! ✅